specification/ai/Face/models.common.tsp

```diff
@@ -0,0 +1,218 @@
+import "@typespec/http";
+import "@typespec/rest";
+import "@azure-tools/typespec-azure-core";
+
+using Azure.Core;
+using TypeSpec.Http;
+using TypeSpec.Rest;
+
+namespace Face;
+
+@doc("The ApiVersion path parameter.")
+model ApiVersionPathParameter {
+  @segment("face")
+  @path("apiVersion")
+  @doc("The API version to use for this operation.")
+  apiVersion: string;
+}
+
+@doc("The request for list resources.")
+model ListRequestOptions {
+  @doc("List resources greater than the \"start\". It contains no more than 64 characters. Default is empty.")
+  @query
+  start?: string;
+
+  @doc("The number of items to list, ranging in [1, 1000]. Default is 1000.")
+  @query
+  @minValue(1)
+  @maxValue(1000)
+  top?: int32;
+}
+
+@doc("The recognition model for the face.")
+enum RecognitionModel {
```

Comment:
extensible enum now need to be written as union

context https://github.com/Azure/azure-rest-api-specs/pull/27687

Reply:
Fixed